packages/grafana-ui/src/components/Segment/Segment.story.tsx

```diff
@@ -37,7 +37,38 @@ SegmentStories.add('Array Options', () => {
             />
             <Segment
               Component={AddButton}
-              onChange={(value: SelectableValue<string>) => action('New value added')(value)}
+              onChange={value => action('New value added')(value as SelectableValue<string>)}
+              options={options}
+            />
+          </div>
+        </>
+      )}
+    </UseState>
+  );
+});
+
+SegmentStories.add('Array Options With Primitive value', () => {
```

Comment:
You can use [CSF](https://storybook.js.org/docs/formats/component-story-format/) instead of this olda way of building stories

Reply:
Nice! I'll look into that.